ConsentViewController/Classes/SourcePointClient/IDFAStatusReportRequest.swift

```diff
@@ -8,8 +8,9 @@
 import Foundation
 
 struct AppleTrackingPayload: Codable {
-    let appleMsgId: Int?
     let appleChoice: SPIDFAStatus
+    let appleMsgId: Int?
+    let partition_uuid: String?
```

Comment:
Similar case here, we can rename the attribute from `partition_uuid` to `messagePartitionUUID`. Since now the attribute will have a different name than the encoded payload, you'll have to override the `CodingKey` ([example](https://github.com/SourcePointUSA/ios-cmp-app/blob/a9eb499f615ec548bf6dc45494444990477e6aa4/ConsentViewController/Classes/SourcePointClient/ConsentRequestResponse.swift#L24))